WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/viewmodel/slice/UiHandler.kt

```diff
@@ -0,0 +1,34 @@
+package org.wordpress.android.ui.posts.prepublishing.home.viewmodel.slice
+
+import androidx.lifecycle.MutableLiveData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.launch
+
+abstract class UiHandler<T : Any> {
```

Comment:
Not sure about this name. I created an abstraction to change handlers on the runtime depending on the FF.

We could use ViewModelSlices instead or take this opportunity to create an abstraction for slices in general. 

The difference here is that this kind of handler receives the instance of the uiState: MutableLiveData<T> that the parent viewModel uses and is able to modify it. This is easier than exposing its own uiState and then merge it back with the other liveData in the parent. But there also downsides, like we could potentially nullify uiState that is shared. 

WDYT?